.lando.yml

```diff
@@ -24,8 +24,8 @@ services:
       - composer install
       - mkdir -p ~/.drush/site-aliases
       - ln -sf /app/drush/nnels-local.aliases.drushrc.php ~/.drush/site-aliases/nnels-local.drushrc.php
-      - drush dl composer_generate
-    run_as_root: updatedb
+      # - drush dl composer_generate
+    # run_as_root: updatedb
```

Comment:
This was the root of the second error:

```
/bin/sh: 1: u: not found
ERROR ==> /bin/sh: 1: u: not found 
```

I quickly googled to find what this command is and didn't come up with anything. I'm assuming this is a custom bash command that you created?

Reply:
updatedb should be provided by mlocate (installed on line 19). I'm using this to find files in the container volumes when I don't know the exact internal path.